scripts/common/rook.sh

```diff
@@ -246,3 +246,33 @@ function rook_operator_ready() {
     fi
     return 0
 }
+
+function rook_is_healthy_to_upgrade() {
+    log "Awaiting to check if rook-ceph ceph is health"
+    if ! $DIR/bin/kurl rook wait-for-health 600 ; then
+        kubectl -n rook-ceph exec deploy/rook-ceph-tools -- ceph status
+        logFail "Failed to detect Rook Ceph is not healthy"
+        return 1
+    fi
+
+    log "Checking Rook versions and replicas"
```

Comment:
I'm confused why you check for this? Doesnt wait-for-health do the same?

Reply:
I am not sure if wait-for-health verify the versions. 
We do those checks for all Rook Upgrades.
Anyway, if wait-for-health does we still without the info in the logs: Example:

<img width="938" alt="Screenshot 2023-03-15 at 17 26 49" src="https://user-images.githubusercontent.com/7708031/225391785-8284e46f-299b-4567-b781-b8a019b2040f.png">

Which are very helpful to us and to the users in order to check what goes wrong.

Reply:
So, WDYT can we move forward with this one?

Reply:
Sorry i was thinking of wait-for-version https://github.com/replicatedhq/kURL/blob/main/pkg/cli/rook_wait_for_version.go

Ok we can move forward.

Reply:
HI @emosbaugh,

wait-for-version has the purpose to ensure a desired versions and because of it requires inform a version to check if that Rook / Ceph version is up. **What version we should pass here?** 

We are checking if the rook installed is OK (Running, Healthy, and not in the knowing scenario of a upgrade failures were we can see that has multiple versions applied)

Also, the wait-for-version is used with the same checks added here in order to provide the helpful outputs see:

https://github.com/replicatedhq/kURL/blob/a341c55ae58d340007ad818123d7f65db2316dc1/addons/rook/1.9.12/install.sh#L281-L294

So, why/how should we use this one?